orderer/common/server/main_test.go

```diff
@@ -73,7 +73,7 @@ func TestInitializeProfilingService(t *testing.T) {
 		l.Close()
 		return l.Addr().String()
 	}()
-	initializeProfilingService(
+	go initializeProfilingService(
```

Comment:
not yours, but i think this go routine has been leaking. probably worth opening a JIRA to clean up this test